src/Microsoft.EntityFrameworkCore.UnitOfWork/IRepository.cs

```diff
@@ -42,15 +43,17 @@ public interface IRepository<TEntity> where TEntity : class
         /// <param name="pageIndex">The index of page.</param>
         /// <param name="pageSize">The size of the page.</param>
         /// <param name="disableTracking"><c>True</c> to disable changing tracking; otherwise, <c>false</c>. Default to <c>true</c>.</param>
+        /// <param name="include">A function to include navigation properties</param>
         /// <returns>An <see cref="IPagedList{TEntity}"/> that contains elements that satisfy the condition specified by <paramref name="predicate"/>.</returns>
         /// <remarks>This method default no-tracking query.</remarks>
-        IPagedList<TEntity> GetPagedList(Expression<Func<TEntity, bool>> predicate = null, Func<IQueryable<TEntity>, IOrderedQueryable<TEntity>> orderBy = null, int pageIndex = 0, int pageSize = 20, bool disableTracking = true);
+        IPagedList<TEntity> GetPagedList(Expression<Func<TEntity, bool>> predicate = null, Func<IQueryable<TEntity>, IOrderedQueryable<TEntity>> orderBy = null, int pageIndex = 0, int pageSize = 20, bool disableTracking = true, Func<IQueryable<TEntity>, IIncludableQueryable<TEntity, object>> include = null);
```

Comment:
We have change to place 'include' after 'orderBy', because the end user either 'GetPagedList()' or 'GetPagedList(predicate: xxx, pageSize: 40)', that's to say, only one case 'GetPagedList(x, y, 0, 30, fase)' is breaking change. So?